rollup.config.js

```
@@ -3,20 +3,44 @@ import resolve from '@rollup/plugin-node-resolve';
+    output: {
+      file: 'dist/react.js',
+      format: 'esm',
+      sourcemap: true,
+    },
+    external: ['react', 'react-dom'],
+    plugins: [typescript({
+      tsconfig: './tsconfig.json',
+      declaration: true,
+      declarationDir: 'dist',
+      rootDir: 'src',
+    }), , resolve(), commonjs()],
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)